### PR TITLE
WPR-8978: Make 'scrollMode' default value as 'native' on Agate components

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [x] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Make `scrollMode` default value as `native` on Scroller, VirtualList, VirtualGridList components

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Scroller, VirtualList and VirtualGridList components now have default `scrollMode` value as `native`.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-8978

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — CHANGES_REQUESTED** [review]

File: VirtualList/tests/useEvent-translate-specs.js [hunk 1]
```diff
@@ -234,6 +234,7 @@ describe('VirtualList useEvent', () => {
 				itemRenderer={renderItem}
 				itemSize={itemSize}
 				onScrollStart={spy}
+				scrollMode="translate"
```
[the remark this answers is not in the document]
Comment (andreitirlalge) [reply VirtualList/tests/useEvent-translate-specs.js:242]: I added scrollMode for all test cases and changed the name of the test suite to mention scrollMode

**Comment by daniel-stoian-lgp** [comment]
please add a changelog
please mark in the pr description that this pr has test cases and that it is a API breaking change

**Review by daniel-stoian-lgp — COMMENTED** [review]
please add a changelog and mark it in the PR description
please mark in the PR description that this PR has test cases and that it is a API breaking change

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 The following is a curated list of changes in the Enact agate module, newest changes on the top.
 
+## [unreleased]
+
+Changed `scrollMode` default value as `native` on the following components: `Scroller`, `VirtualList` and `VirtualGridList`.
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:7]:
```suggestion
- `agate/Scroller`, `agate/VirtualList` and `agate/VirtualGridList` default value for `scrollMode` to `native`.
```

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by dongsuwon — COMMENTED** [review]
I think it's better to remove 'scrollMode=native' in native scroller/virtualList/VirtualGridList unit tests. There are unnecessary codes as changed default value.

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**